### ISSUE TITLE
No IOOS responseFormat is available for some ObservationOffering elements

### ISSUE THREAD

**Issue by abirger** [asascience-open/ncSOS#166]
The `responseFormat` element(s) for each ObservationOffering shall have a value `text/xml; subtype=“om/1.0.0/profiles/ioos_sos/1.0”`; however, in some cases the ncSOS does not describe that required value. 
For example, the [CariCOOS' SWAN GetCapabilities](http://dm2.caricoos.org/thredds/sos/content/swan/Historical/201502260000.nc?service=SOS&request=GetCapabilities&acceptedVersions=1.0.0) does not report the required `responseFormat`, and each ObservationOffering contains only the string

```
<sos:responseFormat>text/xml;subtype="om/1.0.0"</sos:responseFormat>
```

**Comment by cheryldmorse** [comment]
@abirger  ncSOS is explicitly not including that responseFormat due to the fact that the SWAN data is a "grid" featureType. Isn't that correct? Don't the Milestone 1.0 templates just apply to the  "point”, “timeSeries”, and “timeSeriesProfile” feature types?

**Comment by kwilcox** [comment]
@cheryldmorse that's correct, this can probably be closed... the response format is showing up correctly when there is an IOOS template defined for the type.

**Comment by cheryldmorse** [comment]
ncSOS can now handle files with featureType=grid #176

**Comment by cheryldmorse** [comment]
ncSOS still does not handle the point feature type

**Comment by cheryldmorse** [comment]
@mwengren - Do we want to update ncSOS to handle the point feature types?

**Comment by mwengren** [comment]
@cheryldmorse Can you remind me what the level of effort is for this?  If it's minimal, I'd say go ahead an implement this.  If not, it may need further discussion.

@dpsnowden @kbailey-noaa @jbosch-noaa any input on this?
